default.ps1

```diff
@@ -78,7 +78,7 @@ task Nuget {
       @("RhinoDSL", "1.0.0"), `
       @("Common.Logging", "3.0.0"), `
       @("Common.Logging.Core", "3.0.0"), `
-      @("FileHelpers", "2.0.0.0") `
+      @("FileHelpers", "3.0.46-beta") `
```

Comment:
This will not work, see error @MarcosMeli :
A stable release of a package should not have on a prerelease dependency. Either modify the version spec of dependency "FileHelpers (= 3.0.46-beta)" or update the version field.